upgrade/helpers/tasks.py

```diff
@@ -345,6 +346,12 @@ def post_upgrade_test_tasks(sat_host, cap_host=None):
         sat_version=sat_version,
         host=sat_host
     )
+    # Execute task for creating latest discovery iso required for unattended
+    #  test
+    execute(
+        get_discovery_image,
```

Comment:
We will also need BeautifulSoup as requirement for this ryt?

Reply:
@san7ket We will get that from https://github.com/SatelliteQE/automation-tools/blob/master/setup.py#L24